doc/03_Documents/01_Editables/40_WYSIWYG.md

```diff
@@ -4,6 +4,16 @@
 
 Similar to Textarea and Input you can use the WYSIWYG editable in the templates to provide rich-text editing. TinyMce is installed by default in our demo. Another editor can be installed via the wysiwyg-events you find in `events.js`
 
+## Use the default editor (TinyMCE)
+
+In Pimcore 11 the default editor changed from CKEditor to TinyMCE and has been moved into [PimcoreTinymceBundle](https://github.com/pimcore/pimcore/blob/11.x/bundles/TinymceBundle/README.md).
+
+Install the bundle via:
+```bash
+bin/console pimcore:bundle:install PimcoreTinymceBundle
```

Comment:
IMHO this seciont is a little redundant

https://github.com/pimcore/pimcore/pull/16677/files#diff-c601cb407d4defaaf4f0ab272abee11dcc09f7984d5f296e824f86e72ed7e46aL29

The link to readme applied to the `TinymceBundle` word, and put the pimcore:bundle:install command in the https://github.com/pimcore/pimcore/blob/11.x/bundles/TinymceBundle/README.md